perllib/FixMyStreet/Cobrand/Bexley/Garden.pm

```diff
@@ -141,6 +141,8 @@ sub _remove_garden_services {
     }
 }
 
+sub _trim { ( $_[0] // '' ) =~ s/^\s+|\s+$//gr }
+
```

Comment:
I think `Utils::trim_text` is the standard utility for this, as a bonus it will also replace multiple spaces/newlines with a single space as well 🙂